CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+# Set current dir
+CU_SET_PATH("CUR_DIR_MODULE" "${CMAKE_CURRENT_LIST_DIR}")
```

Comment:
i think should not be used CUR_DIR_MODULE here.
CU_SET_PATH is setting global.
CUR_DIR_MODULE should be changed to a unique name.
I see you use the same CUR_DIR_MODULE in other modules.It may be assigned a value repeatedly.